### DISCUSSION TITLE
Any suggestion about model ensemble for mask rcnn?

### DISCUSSION THREAD

**Discussion by animebing** [open-mmlab/mmdetection#8728] [General] [answered]
I have trained several mask rcnn models with different backbones, then I want to ensemble these models for a better performance, but I am totally confused about how to do this for instance segmentation or object detection, can you give me some suggestion or reference for  this? Thank you anyway.

**Comment by Ezra-Yu** [comment]
I have the same problem.
did you slove it?

**Comment by Hesene** [comment]
I have the same problem. did you slove it?

**Comment by harshgrovr** [comment]
I did guess no one did?

**Comment by Etoye** [comment]
Hey. Mosty simple way is to combine all outputs and find a weighted average of it.

**Comment by kentaroy47** [comment, marked as answer]
For boxes ensemble, you can simply nms the multi-model box outputs.

See WBF for references.

https://github.com/ZFTurbo/Weighted-Boxes-Fusion
